CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,8 @@
+This project follows the [Google Open Source Community Guidelines](https://opensource.google/conduct).
```

Comment:
Does this need to change to markdown? (seems unrelated to the PR current description)

Reply:
I thought markdown would be more readable, but actually RST does look nicer.